routes/api/v1.php

```diff
@@ -48,6 +50,7 @@
     Route::put('projects/{projectId}', UpdateStudentProjectController::class)->name('student.updateproject');
     Route::put('skills', UpdateStudentSkillsController::class)->name('student.skills');
     Route::put('profile', UpdateStudentProfileController::class)->name('student.updateProfile');
+    Route::post('languages', AddStudentLanguageController::class)->name('student.addlanguage');
```

Comment:
To keep coherence with the rest of the code that name should be addLanguage in camel case.

Reply:
done